src/extra/AutoUI/index.tsx

```diff
@@ -303,11 +303,13 @@ export const AutoUI = <T extends AutoUIBaseResource<T>>({
 												views={views}
 												autouiContext={autouiContext}
 												changeFilters={(updatedFilters) => {
-													setFilters(updatedFilters);
-													if (onChange) {
-														onChange({
-															filters: updatedFilters,
-														});
+													if (!isEqual(updatedFilters, filters)) {
```

Comment:
How about moving this equality check in the `onFiltersUpdate?.(storedFilters);` useEffect?
If it works then imo it's a more appropriate place to reduce unnecessary changes.

Reply:
Seems to work